.github/workflows/issue.yaml

```diff
@@ -20,6 +20,10 @@ jobs:
         with:
           egress-policy: audit
 
+      - uses: actions/checkout@v4
+        with:
+          sparse-checkout: .github
```

Comment:
👀

Reply:
Yep it needs the issue templates to do the labeling, I can throw in a comment. The label actions do have some magic to fetch the config if it's not available locally, but while we're at it we might as well check out all of `.github` to avoid the extra fetch.

Reply:
Comment added in https://github.com/backstage/backstage/pull/29677/commits/d868732710f37afba382f31a8a43c856da077882